### DISCUSSION TITLE
v2.18.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#7320] [Releases & Announcements]
## Changelog

- New attachment view for conversations
- Ability to export contacts
- Support for interactive messages in WhatsApp ( button and list )
- Ability to update bot/user avatars from Super Admin Dashboard
- Audit logs for `sign_in`, `sign_out`, team events (EE)
- Support for configuring subdomains in widget SDK
- Enhanced filters for CSAT responses
- Support for super script tag in help center articles
- Numerous bug fixes and enhancements

Thanks to @clairton, @steffenhaak, @amplitudesxd, @marcelstoer, @memorycancel, @ChamathKB  for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v2.18.0'>v2.18.0</a>.</em>

**Comment by hasanbasri1993** [comment]
![image](https://github.com/chatwoot/chatwoot/assets/6375150/8831d2ec-b67a-4546-889c-1091a3ff7ebe)
After update i'm getting error

**Reply by sojan-official** [reply]
@hasanbasri1993 
- did you run the migrations ?
- did you restart the servers after running the migrations ?

**Reply by hasanbasri1993** [reply]
i run this
update and migration on docker

````
cd /home/chatwoot/chatwoot;
docker-compose pull;
docker-compose up -d;
docker-compose run --rm rails bundle exec rails db:chatwoot_prepare;
docker-compose run --rm rails bundle exec rake db:migrate
docker-compose up -d
````

getting error
````
{
    "error": "undefined method `processed_message_content' for #<Message id: nil, content: \"Hello fghfghfg\", account_id: 2, inbox_id: 50, conversation_id: 10063, message_type: \"incoming\", created_at: nil, updated_at: nil, private: false, status: \"sent\", source_id: nil, content_type: \"text\", content_attributes: {}, sender_type: \"Contact\", sender_id: 23204, external_source_ids: {}, additional_attributes: {}>"
}
````


success after run, and delete index duplicate first
````
docker exec -it $(basename $(pwd))-rails-1 sh -c 'RAILS_ENV=production bundle exec rails db:chatwoot_prepare'
````

**Comment by glauberssoutodossantos** [comment]
Hi @sojan-official ,

How can I test this? 

> Support for interactive messages in WhatsApp ( button and list )

I just by API?

**Reply by sojan-official** [reply]
@glauberssoutodossantos yes, you need to create interactive messages in a conversation using the APIs.

**Reply by Double-Click-ERP** [reply]
any example for the API body ?

**Reply by sojan-official** [reply]
https://www.chatwoot.com/docs/product/others/interactive-messages#1-options

**Reply by Double-Click-ERP** [reply]
Thank You

**Comment by sojan-official** [comment]
## 2.18.0 Dev Changelog

### API Endpoints

#### Additions
- `/api/v1/accounts/{account.id}/contacts/exports` [POST]
  - https://github.com/chatwoot/chatwoot/pull/7258
  - Endpoint to trigger contact export emails
  
#### Deletions
- none

---

### Other Changes
- none

**Reply by mssmello** [reply]
Hi @sojan-official , 

If I install a new 2.18.0 it´s all fine. 
When I upgraded my 2.16.0 to 2.18.0 using cwctl --upgrade, I cant see Ability to export contacts .
Is there any cache to clear? Thanks.

**Reply by mssmello** [reply]
Oppss! 
Sorry,, it´s all right. I can see the Export Contacts at my upgraded version from 2.16.0 to 2.18.0. 
BR.

**Comment by cruz4d3r** [comment]
Hi guys, i have a problem with a fresh installation, when try filter the contacts. Dont work.
